assets/js/pwa/sw.js

```diff
@@ -84,7 +84,7 @@ self.addEventListener('fetch', (event) => {
       return fetch(event.request).then((response) => {
         const url = event.request.url;
 
-        if (purge || event.request.method !== 'GET' || !verifyUrl(url)) {
+        if (purge || response.status !== 200 || event.request.method !== 'GET' || !verifyUrl(url)) {
```

Comment:
Changing the condition to hit `206` to return the response directly would restore the functionality of the PWA, but I'm not sure that's the best way to do it

```suggestion
        if (purge || response.status == 206 || event.request.method !== 'GET' || !verifyUrl(url)) {
```

Reply:
any thoughts?